src/client/components/footer/__snapshots__/Footer.test.js.snap

```diff
@@ -0,0 +1,221 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Footer> Snapshot +++capturing snapshot of Footer 1`] = `
```

Comment:
this is not informative. I would see the dom tree, not a react childs

Reply:
Dom tree created in snapshot